docs/sdk.md

```diff
@@ -16,33 +16,55 @@ Should you find bugs or in the case you need help please reach out to us in [Dis
 
 To get you started here's how you would instantiate the SDK and fetch some NFTs
 
-1. Install the sdk
+1. Install the sdk and other dependencies
 
 ```shell
-npm install @3rdweb/sdk
+npm install @3rdweb/sdk ethers dotenv
 ```
-2. Get your NFT contract address from the [Admin Dashboard](https://thirdweb.com/dashboard)<!-- -->.
+2. Create a .env file to save your PRIVATE_KEY , you can check how to get your wallet's private key, on [this guide](https://thirdweb.com/portal/guides/create-a-metamask-wallet#export-your-private-key). Also add this .env file to .gitignore.

```

Comment:
remove the suggestion of using `.env` for private key. even though it's convenient it's generally a security risk because it's a known path and can be easily targeted by OS-level file crawler. 
it might be better to put a placeholder below and instruct user to keep it save and make sure to not commit it onto github.

Reply:
Sure, changing the instruction.